.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+# Only triggers on pushes/PRs to master
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+      - github_actions
+
+jobs:
+    test:
+        name: Test Suite
+        strategy:
+            matrix:
+                os: [ubuntu-latest, windows-latest, macOS-latest]
+                dc: [dmd-latest, ldc-latest]
+                arch: [x86_64]
+                include:
+                  - { config:select }
+                  - { config:epoll }
+                  - { config:cfrunloop }
```

Comment:
Github CI, unlike Travis, is does not allow you to add rows to the matrix. So this probably won't work.

Reply:
You want:
```yaml
- { os: whatever, config: name }
```

For all three OSes

Reply:
I've gone the other way around now and used `config: [select, epoll, cfrunloop]`, with an `exclude:` to remove bad configurations, which seems to work right. Just Windows doesn't produce any test output.